fireapp/lib/presentation/shift_request/ShiftRequestWidget.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+class ShiftRequestWidget extends StatelessWidget {
+  final String? title;
+  final String? content;
+
+  const ShiftRequestWidget({Key? key, this.title, this.content}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              title ?? '',
+              style: Theme.of(context).textTheme.headline6,
+              textAlign: TextAlign.left,
+            ),
+            const SizedBox(width: 24,),
```

Comment:
This is a nitpick over code I think this is temporary, but you shouldn't use magic number distances. Express them as <x>.rdp(), where x is multiplied by 16. For this example, you would want `1.5.rdp()`.